src/frontends/tensorflow/src/utils.hpp

```diff
@@ -175,8 +175,9 @@ void values_from_const_node(const NodeContext& node, ov::Shape* const_tensor_sha
                 FRONT_END_THROW("Encountered unknown element type " + DataType_Name(dt) + " on an empty tensor_proto");
             }
             TENSORFLOW_OP_VALIDATION(node, val_size != 0, "Empty values vector");
-
-            if (i < val_size) {
+            if (val_size == 0) {
```

Comment:
I looks really odd, if val_size is actually zero, what then is the type of values in constant?